### PR TITLE
Added initial data and processing notebooks.

### PR DESCRIPTION
**Pull request by DmitryKutsev**
* Added data with the info about Indian companies for SQL source
* Added data with Job descriptions for Vector Storage
* Added notebooks for processing the data

### CODE REVIEW

**Review by NemoLeerink — APPROVED** [review]
Look good :) Only I don't understand in process_faiss why 'full stack' is removed from the string (it only occurs in 17 entries), why only the first 20 entries are included and why length has to be smaller than 200.

**Review by NemoLeerink — APPROVED** [review]

**Review by NemoLeerink — APPROVED** [review]

File: data/process_faiss.ipynb [hunk 1]
```diff
@@ -0,0 +1,277 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain.embeddings.openai import OpenAIEmbeddings\n",
+    "from langchain.text_splitter import CharacterTextSplitter\n",
```
Comment (DmitryKutsev) [inline data/process_faiss.ipynb:10]: Test comment

**Comment by DmitryKutsev** [comment]
> > Look good :) Only I don't understand in process_faiss why 'full stack' is removed from the string (it only occurs in 17 entries), why only the first 20 entries are included and why length has to be smaller than 200.
> 
> I removed it to check how the model could rank descriptions with the query "Full stack" without mentioning it at all.

Ah and only 20 entries because I was just testing if it works or not and didn't want to spend money on it with ChatGPT (also haven't calculated though, but just in case)

**Comment by NemoLeerink** [comment]
Makes sense, thank you!

**Pull request merged**